app/models/brand.rb

```diff
@@ -82,6 +82,12 @@ def twitter_id
     twitter_identity.uid.to_i
   end
 
+  def tweet_url(tweet)
```

Comment:
Ah, so we don't need to store the url. I see you weren't. Just the post date and the id.